test/Microsoft.ApplicationInsights.AspNet.Tests/project.json

```diff
@@ -1,26 +1,23 @@
 {
   "dependencies": {
-    "Microsoft.ApplicationInsights.AspNet": "0.32.0-beta4",
-    "Microsoft.AspNet.FeatureModel": "1.0.0-beta4-*",
-    "Microsoft.AspNet.Http": "1.0.0-beta4-*",
-    "Microsoft.AspNet.Http.Extensions": "1.0.0-beta4-*",
-    "Microsoft.AspNet.Hosting": "1.0.0-beta4-*",
-    "Microsoft.AspNet.Mvc.Core": "6.0.0-beta4-*",
-    "Microsoft.Framework.Logging": "1.0.0-beta4-*",
-    "Microsoft.Framework.ConfigurationModel.Json": "1.0.0-beta4-*",
-    "xunit": "2.1.0-*",
-    "xunit.runner.dnx": "2.1.0-*",
-    "xunit.runner.visualstudio": "2.1.0-*"
+    "Microsoft.ApplicationInsights.AspNet": "1.0.0-beta5",
+    "Microsoft.AspNet.FeatureModel": "1.0.0-beta5-*",
+    "Microsoft.AspNet.Http": "1.0.0-beta5-*",
+    "Microsoft.AspNet.Http.Extensions": "1.0.0-beta5-*",
+    "Microsoft.AspNet.Hosting": "1.0.0-beta5-*",
+    "Microsoft.AspNet.Mvc.Core": "6.0.0-beta5-*",
+    "Microsoft.Framework.Logging": "1.0.0-beta5-*",
+    "Microsoft.Framework.Configuration.Json": "1.0.0-beta5-*",
+    "xunit.runner.aspnet": "2.0.0-aspnet-*"
   },
   "commands": {
-    "test": "xunit.runner.dnx"
+    "web": "Microsoft.AspNet.Hosting --server Microsoft.AspNet.Server.WebListener --server.urls http://localhost:5000",
+    "gen": "Microsoft.Framework.CodeGeneration",
+    "ef": "EntityFramework.Commands",
```

Comment:
do we need the web, gen and ef commands? Are we referencing the `Microsoft.Framework.CodeGeneration` and `EntityFramework.Commands` packages where the `gen` and `ef` commands are defined?